<launch>
    <!-- Run on PC -->

    <!-- end effector interactive goal marker -->
    <node pkg="mars_mot_plan" type="ee_goal_marker_node" name="ee_goal_marker_node" output="screen">
        <param name="world_frame" value="vive_world"/>
    </node>
    
</launch>